src/istgtcontrol.c

```diff
@@ -1066,6 +1066,44 @@ exec_dump(UCTL_Ptr uctl)
 	return UCTL_CMD_OK;
 }
 
+static int
+exec_iostats(UCTL_Ptr uctl)
+{
+        const char *delim = ARGS_DELIM;
+        char *arg;
+        char *result;
+        int rc;
+
+        uctl_snprintf(uctl, "IOSTATS\n");
+        rc = uctl_writeline(uctl);
+        if (rc != UCTL_CMD_OK) {
+                return rc;
+        }
+        /* receive result */
+	while(1) {
+		rc = uctl_readline(uctl);
+		if (rc != UCTL_CMD_OK) {
+			return rc;
+		}
+		arg = trim_string(uctl->recvbuf);
+		result = strsepq(&arg, delim);
+		strupr(result);
+		if (strcmp(result, uctl->cmd) !=0){
+			break;
+		}
+		if (uctl->iqn != NULL) {
+			printf("%s\n", arg);
+		} else {
+			printf("%s\n", arg);
```

Comment:
here, you are printing arg in both the cases. What is the difference here?
should we handle IQN number?

Reply:
this is the case with the every command in the istgtcontrol. Anyway i'm removing it.